package.json

```diff
@@ -37,8 +37,9 @@
     "nouislider": "^13.1.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-redux": "^9.0.4",
     "react-i18next": "^13.5.0",
+    "react-redux": "^9.0.4",
+    "react-responsive": "^9.0.2",
```

Comment:
Not needed anymore I believe? :) To remove the dependencies remove this line and re-install libraries to see the change in yarn.lock, please.